src/Persister/converters/PasswordPolicyEntityConverter.ts

```diff
@@ -0,0 +1,26 @@
+import { User } from "../../GSuite/GSuiteClient";
+import {
+  PASSWORD_POLICY_ENTITY_CLASS,
+  PASSWORD_POLICY_ENTITY_TYPE,
+  PasswordPolicyEntity
+} from "../../JupiterOne";
+
+export function generatePasswordPolicyId(userId?: string) {
+  return `gsuite-password-policy-user-id-${userId}`;
```

Comment:
I think the `id-` bit can be dropped from these prefixes since they don't add any information.

```suggestion
  return `gsuite-password-policy-user-${userId}`;
```